src/silo_api/query_handler.cpp

```diff
@@ -29,10 +33,11 @@ void QueryHandler::post(
    SPDLOG_INFO("received query: {}", query);
 
    response.setContentType("application/json");
-
    try {
       const auto query_result = query_engine.executeQuery(query);
 
+      response.set("data-version", data_version);
```

Comment:
Is it possible to test this in the `request_handler_factory.test.cpp`?

Reply:
Done :)